lifecycle-operator/apis/lifecycle/v1alpha3/keptntaskdefinition_types.go

```diff
@@ -55,6 +55,11 @@ type KeptnTaskDefinitionSpec struct {
 	// +kubebuilder:validation:Type:=string
 	// +optional
 	Timeout metav1.Duration `json:"timeout,omitempty"`
+	// Service Account to be used in jobs to authenticate with the Kubernetes API and access cluster resources.
```

Comment:
To stay in line with the Golang comment conventions, please start the comment document each property with exactly the name of the property, i.e. something like:

```suggestion
	// ServiceAccount specifies the service account to be used in jobs to authenticate with the Kubernetes API and access cluster resources.
```

Also, to tidy up the documentation, we should consistently use one way of writing "service account" - right now we switch between "serviceAccount" and "service account". I suggest to use "service account".